### ISSUE TITLE
Error when make install 

### ISSUE THREAD

**Issue by HengCC** [apitable/apitable#531]
code branch: 0.18.0-rc1

execute `make install` .  it will execute 2 commands first `yarn install && yarn build:pre`, it throw the error below when execute the second command

![image](https://user-images.githubusercontent.com/6167532/223939813-ee5f95dc-7c53-40f4-9d20-798426617abf.png)

**Comment by arucil** [comment]
Its network problem, make sure you are using a network proxy in the terminal.

**Comment by HengCC** [comment]
@arucil   I can confirm that the network is fine, All requests go through the proxy. 
![image](https://user-images.githubusercontent.com/6167532/223946959-0d288053-eead-40ab-b80c-804ebb6c6ba9.png)

![image](https://user-images.githubusercontent.com/6167532/223946306-378ff099-a85e-4c3e-b7ad-a5e7b2da4db0.png)

**Comment by arucil** [comment]
Try setting `https_proxy` and `http_proxy` env vars to your proxy.

**Comment by HengCC** [comment]
> Try setting `https_proxy` and `http_proxy` env vars to your proxy.

yes, i did that .  export https_proxy=http://127.0.0.1:7890 http_proxy=http://127.0.0.1:7890 all_proxy=socks5://127.0.0.1:7890

**Comment by HengCC** [comment]
The final solution is to upgrade cargo, because of the difference between the Chinese documentation (there is no step to install rust) and the English documentation. As a result, I did not notice the need to install rust. I already had rust on my computer. So in the initial version I compiled successfully. But after pulling the latest code, the compilation kept failing. I didn’t realize the version difference until I saw the English document that needed to be installed. My local cargo is around January 2022.

However, the Chinese documentation can also be updated.
